notebooks/ROS 3D Grid.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": null,
```

Comment:
I think you can revert this file, and some of the other notebooks so that they do not appear in the diff.